Dockerfile

```
@@ -17,24 +17,24 @@ RUN apt-get update && \
 RUN add-apt-repository -y ppa:ondrej/php && \
     apt-get update
 
-# Install PHP 8.2 and required extensions (matching composer.json requirement)
+# Install PHP 8.4 and required extensions (matching composer.json requirement)
 RUN apt-get install -y \
```

Hadolint (2.14.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)